checks.go

```diff
@@ -21,16 +21,18 @@ type Check struct {
 	Port      int    `json:"msp_port,omitempty"`
 	IPVersion string `json:"msp_use_ip_version,omitempty"`
 
-	Interval    int      `json:"msp_interval,omitempty"`
-	Locations   []string `json:"locations,omitempty"`
-	Sensitivity int      `json:"msp_sensitivity,omitempty"`
-	Threshold   int      `json:"msp_threshold,omitempty"`
-
-	Headers      string `json:"msp_headers,omitempty"`
-	Username     string `json:"msp_username,omitempty"`
-	Password     string `json:"msp_password,omitempty"`
-	SendString   string `json:"msp_send_string,omitempty"`
-	ExpectString string `json:"msp_expect_string,omitempty"`
+	Interval        int      `json:"msp_interval,omitempty"`
+	Locations       []string `json:"locations,omitempty"`
+	Sensitivity     int      `json:"msp_sensitivity,omitempty"`
+	Threshold       int      `json:"msp_threshold,omitempty"`
+	NumRetries      int      `json:"msp_num_retries,omitempty"`
+	UptimeSLA       int      `json:"msp_uptime_sla,omitempty"`
```

Comment:
SLAs are fixed-point decimals serialized into strings. Int won't work out here.

Reply:
Thank you for your comment, I will update the code and send it

Reply:
I have updated type of ResponseTImeSLA based on your suggestion, can you check it?

Reply:
@mikluko

Reply:
I'm afraid I can't approve it as it is. The value is a decimal field _encoded_ into a string. Not literal string. It should be backed by some implementation capable of decoding incoming string into a decimal value. 

There is a pre-release of this library v2. It probably addresses your issue, please have a look: https://github.com/uptime-com/uptime-client-go/blob/56f57032586c79e75ffc22062520648f62f9a760/pkg/upapi/ep_checks.go#L67-L68

Reply:
Indeed, it would be excellent to make this version available promptly, allowing us to meet our team's requirements and continue our work on Terraform seamlessly.